docker/Dockerfile.alpine

```diff
@@ -1,40 +1,26 @@
-FROM alpine:3.8
+FROM alpine:3.11
+RUN echo 'http://dl-cdn.alpinelinux.org/alpine/v3.10/main' >> /etc/apk/repositories
+RUN echo 'http://dl-cdn.alpinelinux.org/alpine/edge/community' >> /etc/apk/repositories
 RUN apk add --update \
   bison \
-  binutils-dev \
+  bcc-dev==0.12.0-r2 \
+  bcc-static==0.12.0-r2 \
+  libbpf-dev \
   build-base \
-  clang-dev \
-  clang-static \
-  curl \
+  clang-dev==8.0.0-r0 \
```

Comment:
Similarly, clang 8 packages are not available in 3.11 or later even though llvm 8 is, so these are fetched from the 3.10 repository until we can find a better solution.